src/Proj4.jl

```diff
@@ -1,6 +1,8 @@
 module Proj4
 
-export transform, transform!, Projection, is_latlong, is_gencent
+import Compat
```

Comment:
Ah, I see the problem you're having below - this should be `using Compat` instead.